src/commands/Unban.ts

```diff
@@ -64,17 +67,33 @@ async function unbanUserFromRooms(
         continue;
       }
       if (rule.test(member.userID)) {
+        let logMessage = `Unbanning ${member.userID} in ${revision.room.toRoomIDOrAlias()}`;
+        if (invite) {
+          logMessage += ", and re-inviting them.";
+        }
         await managementRoomOutput.logMessage(
           LogLevel.DEBUG,
           "Unban",
-          `Unbanning ${member.userID} in ${revision.room.toRoomIDOrAlias()}`,
+          logMessage,
           revision.room.toRoomIDOrAlias()
         );
         if (!noop) {
           await roomUnbanner.unbanUser(
             revision.room.toRoomIDOrAlias(),
             member.userID
           );
+          const inviteResult = await roomInviter.inviteUser(
+            revision.room,
+            member.userID
+          );
+          if (isError(inviteResult)) {
+            await managementRoomOutput.logMessage(
+              LogLevel.WARN,
+              "Unban",
+              `Failed to re-invite ${member.userID} to ${revision.room.toRoomIDOrAlias()}`,
+              revision.room.toRoomIDOrAlias()
+            );
+          }
```

Comment:
So when I ran this locally, for some reason this failed every single time, and the reason was that there was a bug in MPS that banned instead of unbanning, that's been fixed now and so we'll have to rebase....